typeform/typeform.py

```diff
@@ -22,9 +21,6 @@ def __init__(self, source, options):
         if 'destination' not in source:
             source['destination'] = DESTINATION
 
-        # append the destination postfix, which represent the form name
-        source['destination'] += DESTINATION_POSTFIX
```

Comment:
@alonbrody Regarding what we talked, after validating, we can't omit the [`source-change` firing](https://github.com/avinoamr/color/blob/master/src/sources-runner/python/runner.py#L30), cause the `writer` and python sources aren't sharing the same processes and therefore we can't exploit the shared memory that happens between the `javascript` sources and the writer. We might want to think for a better solution in the future..  CC: @avinoamr